.tool-versions

```diff
@@ -1 +0,0 @@
-golang 1.16.2
```

Comment:
asdf will look at the go version in the go.mod file, no need to keep a tool-versions